msm/src/fec/interpreter.rs

```diff
@@ -583,11 +511,22 @@ pub fn ec_add_circuit<
             F::from_biguint(&carry_f).unwrap()
         };
 
-        let assign_carry = |env: &mut Env, newcarry: F, carryvar: &mut F, extra_offset: usize| {
+        fn assign_carry<F, Env, Foo>(
```

Comment:
Foo -> `ColumnMapping`? or something close to the description?

Reply:
Yep, renamed into `ColMap`, much better than `Foo` :)